unit-tests/fortran_labelled_exit_test.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+##############################################################################
+# (c) Crown copyright 2018 Met Office. All rights reserved.
+# The file LICENCE, distributed with this code, contains details of the terms
+# under which the code may be used.
+##############################################################################
+"""
+Test of the rule for missing exit labels.
+"""
+from typing import List
+
+import pytest  # type: ignore
+# ToDo: Obviously we shouldn't be importing "private" modules but until pytest
+#       sorts out its type hinting we are stuck with it.
+#
+from _pytest.fixtures import FixtureRequest  # type: ignore
+
+import stylist.fortran
```

Comment:
Lets be specific about what we are importing.

Reply:
fixed.